sdk-extensions/autoconfigure/src/testFullConfig/java/io/opentelemetry/sdk/autoconfigure/ConfigurableSpanExporterTest.java

```diff
@@ -10,59 +10,146 @@
 
 import com.google.common.collect.ImmutableMap;
 import io.opentelemetry.exporter.logging.LoggingSpanExporter;
+import io.opentelemetry.exporter.otlp.trace.OtlpGrpcSpanExporter;
 import io.opentelemetry.exporter.zipkin.ZipkinSpanExporter;
 import io.opentelemetry.sdk.trace.export.BatchSpanProcessor;
 import io.opentelemetry.sdk.trace.export.SimpleSpanProcessor;
 import io.opentelemetry.sdk.trace.export.SpanExporter;
 import java.util.Collections;
 import java.util.Map;
+import org.assertj.core.api.InstanceOfAssertFactories;
 import org.junit.jupiter.api.Test;
 
 public class ConfigurableSpanExporterTest {
 
   @Test
-  void configuration() {
+  void configureSpanExporters_spiExporter() {
     ConfigProperties config =
-        DefaultConfigProperties.createForTest(ImmutableMap.of("test.option", "true"));
-    SpanExporter spanExporter = SpanExporterConfiguration.configureExporter("testExporter", config);
+        DefaultConfigProperties.createForTest(
+            ImmutableMap.of("test.option", "true", "otel.traces.exporter", "testExporter"));
+    Map<String, SpanExporter> exportersByName =
+        SpanExporterConfiguration.configureSpanExporters(config);
 
-    assertThat(spanExporter)
+    assertThat(exportersByName)
+        .hasSize(1)
+        .containsKey("testExporter")
+        .extracting(map -> map.get("testExporter"))
         .isInstanceOf(TestConfigurableSpanExporterProvider.TestSpanExporter.class)
         .extracting("config")
         .isSameAs(config);
   }
 
+  @Test
+  void configureSpanExporters_duplicates() {
+    ConfigProperties config =
+        DefaultConfigProperties.createForTest(
+            ImmutableMap.of("otel.traces.exporter", "otlp,otlp,logging"));
+
+    assertThatThrownBy(() -> SpanExporterConfiguration.configureSpanExporters(config))
+        .isInstanceOf(ConfigurationException.class)
+        .hasMessageContaining("otel.traces.exporter contains duplicates: [otlp]");
+  }
+
+  @Test
+  void configureSpanExporters_multipleWithNone() {
+    ConfigProperties config =
+        DefaultConfigProperties.createForTest(ImmutableMap.of("otel.traces.exporter", "otlp,none"));
+
+    assertThatThrownBy(() -> SpanExporterConfiguration.configureSpanExporters(config))
+        .isInstanceOf(ConfigurationException.class)
+        .hasMessageContaining("otel.traces.exporter contains none along with other exporters");
+  }
+
```

Comment:
good tests. Do we have a test for something silly like a single comma (or several commas)? I would hope that would fail somehow, but I didn't see in the code how it might fail.

Reply:
That situation isn't possible, since `DefaultConfigProperties#getCommaSeparatedValues(..)` filters out blank and null entries. 

I guess the question is whether we trust the responses of `DefaultConfigProperties#getCommaSeparatedValues(..)` to do what the javadoc says, or need to add additional checks / tests. My inclination is to not add additional checks / tests, since there is a test in `ConfigPropertiesTest` that confirms this already. 
```
Returns a list-valued configuration property. The format of the original value must be comma-separated. Empty values will be removed.
```